### DISCUSSION TITLE
Incentivize Community Engagement

### DISCUSSION THREAD

**Discussion by JustMaier** [civitai/civitai#111] [Ideas]
Creators are the lifeblood of Civitai, and good reviews and community engagement are the fuel that keeps creators creating. To encourage community engagement, we'd like to find ways to incentivize members of the community to leave reviews, make comments, respond to comments, and react to awesome reviews and funny comments.

## Here are our current ideas
- Remind people to review models that they've downloaded by sending them a notification 3 days after downloading something prompting them to share their thoughts on the model. This notification would link directly to the leave a review modal.
- Give users a download history page so they can go back and review what they've downloaded and see whether or not they've left a review. Allow them to also hide/remove things from their history like Amazon order history.
- Award trophies (details for those below)
- Add trophy booth to user page so that people can show off their accolades

## Trophies
- **Curator:** Awarded for reviewing models. Wood (1), Bronze (5), Silver (15), Gold (50), Platinum (100)
- **Creator:** Awarded for uploading models (must receive 100 downloads to count). Wood (1), Bronze (2), Silver (5), Gold (15), Platinum (50)
- **Popcorn:** Awarded for reacting to comments and reviews. Wood (1), Bronze (10), Silver (25), Gold (100), Platinum (250)
- **Craftsman:** Award for each model you upload that receives over 5 5-star reviews. Wood (1), Bronze (2), Silver (5), Gold (15), Platinum (50)
- **Critic:** Awarded for each review you make that receives over 5 reactions. Wood (1), Bronze (5), Silver (15), Gold (50), Platinum (100)
- **Trend-setter:** Awarded for each day one of your models is in the top 10 downloads of the day. Wood (1), Bronze (3), Silver (7), Gold (30), Platinum (90)
- **Troublemaker:** Awarded for having a review or model that has received a minimum of 10 interactions that has a maximum of a 60/40 split between 1 star and 5 star reviews (thumbs up thumbs down for comments)
- **Distributor:** awarded for total downloads across all models: Wood (2000) Bronze (6000) Silver (10000) Gold (18000) Platinum (25000)
- **Trashcan:** awarded for downloading models with less than 2 star reviews (models with no stars do not count): Wood (10) Bronze (50) Silver (100) Gold (200) Platinum (500)
- **Explorer:** Awarded for downloading models with no reviews Wood: (10) Bronze (50) Silver (100) Gold (200) Platinum (500)
- **Trailblazer:** awarded for leaving the first review on a model with images: Wood (1) Bronze (10) Silver (50) Gold (100) Platinum (250)
- **Lurker** Awarded for each day that you downloaded at least 1 model, but did no engagement: wood (10) Bronze (50) Silver (100) Gold (200) Platinum (500)
- **Voyeur** Awarded for consecutive days a user has existed without doing any action (downloading included): Wood (10) Bronze (50) Silver (100) Gold (200) Platinum (500) Trophy is removed after any action
- **Architect** Awarded for updating a model (adding a new version) Wood (1) Bronze (5) Silver (20) Gold (50) Platinum (100)
- **Collector** Awarded for downloading unique models Wood (10) Bronze (50) Silver (100) Gold (200) Platinum (500)

## TROPHIES PENDING NEW FEATURES
- **Knowledge Seeker:** Awarded for asking a question in the Q&A section
- **Sherpa:** Awarded for answering unique question in the Q&A section: Wood (1) Bronze (5) Silver (10) Gold (20) Platinum (50)
- **Civit** Awarded for joining the discord
- **Hunter** Awarded for submiting a bounty: Wood (1) Bronze (5) Silver (10) Gold (20) Platinum (50)
- **Seeker** Awarded for requesting a bounty: Wood (1) Bronze (5) Silver (10) Gold (20) Platinum (50)
- **Smoothest of brains** Awarded for having a platinum in all trophies (other then Lurker and Voyeur)

**Comment by Janca** [comment]
I also think having a gallery of all the generations a user has submitted for any model available on their profile for the community to vote directly, and back-linking the model with the `i` or some other way would be a good idea, maybe sort the gallery by rank or newest, w.e

**Reply by Janca** [reply]
Just a continuation, but if galleries become a thing, I think it would also be nice to allow a user to feature an `x` amount of creations that are featured above all their others, to allow the user to highlight what they think is their best creation.

**Comment by RyanGin0** [comment]
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
